app/main/views/suppliers.py

```diff
@@ -0,0 +1,23 @@
+from flask import jsonify
+
+from .. import main
+from ...models import Supplier
+
+
+@main.route('/suppliers/<int:supplier_id>', methods=['GET'])
+def get_supplier(supplier_id):
+    supplier = Supplier.query.filter(
+        Supplier.supplier_id == supplier_id
+    ).first_or_404()
+
+    return jsonify(suppliers=jsonify_supplier(supplier))
+
+
+def jsonify_supplier(supplier):
+    data = {
+        'id': supplier.id,
```

Comment:
Good spot, yes we shouldn't expose the database id at all.  I think we want `'id': supplier.supplier_id` and no `supplier_id` key in the JSON.